Library/Formula/fish.rb

```diff
@@ -11,16 +14,12 @@ class Fish < Formula
 
   head do
     url "https://github.com/fish-shell/fish-shell.git", :shallow => false
-
-    depends_on "autoconf" => :build
-    # Indeed, the head build always builds documentation
-    depends_on "doxygen" => :build
   end
 
   skip_clean "share/doc"
 
   def install
-    system "autoconf" if build.head?
+    system "autoconf"
```

Comment:
Sure thing. (I was on the fence about whether that info belonged in a comment, in the commit message, or in the PR discussion.)